src/main/java/codeu/controller/ParsingEngine.java

```diff
@@ -0,0 +1,43 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+import codeu.model.data.BotStructure;
+import codeu.model.data.Message;
+import codeu.model.data.RoutingEngine;
+
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+	
+
+public class parsingEngine {
```

Comment:
Please add a doc string about `ParsingEngine`. What should it be used for? How should it be invoked?

Additionally, nothing is calling into `ParsingEngine` yet. Is the plan to make another PR that calls `ParsingEngine.parseCommands(message)` every time a message is added to the data store or somewhere else?